config/standalone-config.properties

```diff
@@ -10,38 +10,64 @@
 
 ##region General Segment Store Settings
 
-# Port on which a ZK server is started
+# Port on which a ZK server is started.
+# Default value: 4000
 #singlenode.zkPort=4000
 
 # Port on which segmentstore server is started.
+# Default value: 6000
 #singlenode.segmentstorePort=6000
 
 # Port on which the controller server grpc is started.
+# Default value: 9090
 #singlenode.controllerPort=9090
 
-# Enable REST server.
-#singlenode.enableRestServer=true
+# Whether to enable REST server.
+# Default value: false
+singlenode.enableRestServer=false
```

Comment:
Is this change related to the original issue? Moreover, this property can be commented as the other ones that do not modify the default value set in `SingleNodeConfig.java`.

Reply:
This was a bug. I have pushed a commit to restore the default back to true in ``SingleNodeConfig.java``.